### PR TITLE
Uzbek layout

### PR DESCRIPTION
**Pull request by tenextractor**
From Gboard
image

### CODE REVIEW

**Review by Julow — COMMENTED** [review]
The layout is 11 columns, which is quite large for some devices. Couldn't oʻ and gʻ be moved in the corner of an other key to make all the keys bigger ?

**Comment by tenextractor** [comment]
It's a copy of AOSP and Gboard layout, and doesn't change what people are used to.

But if needed we can change it to this:
```
q w e r t y u i o p
a s d f g h j k l ʻ (U+02BB turned comma)
z x c v b n m ʼ (U+02BC modifier letter apostrophe)
```

**Comment by Julow** [comment]
Less wide would definitely be better.

**Comment by tenextractor** [comment]
image

**Review by Julow — APPROVED** [review]
Thanks! Let's merge.

**Review by Julow — COMMENTED** [review]
Many thanks for your great contributions. You are based on an old commit. Could you make sure to pull the master branch before creating new branches in the future ?

File: res/values/layouts.xml [hunk 1]
```diff
@@ -61,6 +61,7 @@
     <item>latn_qwerty_sr</item>
     <item>latn_qwerty_tly</item>
     <item>latn_qwerty_tr</item>
+    <item>latn_qwerty_uz</item>
```
[the remark this answers is not in the document]
Comment (tenextractor) [reply res/values/layouts.xml:64]: Okay

**Pull request merged**